android/app/build.gradle

```diff
@@ -31,6 +31,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
```

Comment:
Ugh, it has to be in the app because WebRTC needs it? That way SDK consumers will need to add it to their apps.

Reply:
It looks that way. I tried to apply it only to the react-native-webrtc module, to no avail.